src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+
+  const LONG_TERM = 7;
+
+  const LONG_TERM_DISCOUNT = 50;
+
+  const SHORT_TERM = 3;
+
+  const SHORT_TERM_DISCOUNT = 20;
+
+  const price = days * PRICE_PER_DAY;
```

Comment:
It is good practice to separate constants and regular variables, and even better to move immutable constants outside the function definition
```suggestion
  const PRICE_PER_DAY = 40;
  const LONG_TERM = 7;
  const LONG_TERM_DISCOUNT = 50;
  const SHORT_TERM = 3;
  const SHORT_TERM_DISCOUNT = 20;

  const price = days * PRICE_PER_DAY;
```